tests/runtests.py

```diff
@@ -356,9 +356,8 @@ def django_tests(verbosity, interactive, failfast, keepdb, reverse,
                  timing, shuffle):
     if verbosity >= 1:
         msg = "Testing against Django installed in '%s'" % os.path.dirname(django.__file__)
-        max_parallel = default_test_processes() if parallel == 0 else parallel
-        if max_parallel > 1:
-            msg += " with up to %d processes" % max_parallel
+        if parallel > 1:
+            msg += " with up to %d processes" % parallel
```

Comment:
Can we restore this message when `auto` or `DJANGO_TEST_PROCESSES` is used? I think it's helpful.

Reply:
I find the message does appear already in these cases:

```
$ ./runtests.py test_runner  --parallel auto
Testing against Django installed in '/Users/chainz/Documents/Projects/django/django' with up to 12 processes
...

$ DJANGO_TEST_PROCESSES=7 ./runtests.py test_runner
Testing against Django installed in '/Users/chainz/Documents/Projects/django/django' with up to 7 processes
...
```

Reply:
Right, sorry. It's missing only without `--parallel` and `DJANGO_TEST_PROCESSES` :detective: :
- on the `main` branch:
  ```
  $ ./runtests.py queries
  Testing against Django installed in '/home/felixx/repo/django/django' with up to 8 processes
  Found 416 test(s).
  Creating test database for alias 'default'...
  Cloning test database for alias 'default'...
  ...
  ```
- with this patch:
  ```
  $ ./runtests.py queries
  Testing against Django installed in '/home/felixx/repo/django/django'
  Found 416 test(s).
  Creating test database for alias 'default'...
  Cloning test database for alias 'default'...
  ```

Can we restore?

Reply:
Done